source/loader/ur_lib.cpp

```diff
@@ -8,10 +8,15 @@
  *
  */
 #include "ur_lib.hpp"
+#include "logger/ur_logger_api.hpp"
 #include "ur_loader.hpp"
 #include "ur_proxy_layer.hpp"
 #include "validation/ur_validation_layer.hpp"
 
+namespace logger {
+Logger logger = create_logger("loader");
```

Comment:
The way we have this right now, this will cause an uncaught exception if something is misconfigured with the logger.
I suggest we catch all the exceptions in `create_logger` and print to stderr if something is wrong. Normally libraries should never print by default, but I guess printing to stderr if the user is trying to set up the logger is reasonable. There's no other simple way to return an error during global initialization.